readme.md

```diff
@@ -2,6 +2,8 @@
 
 > Convenience wrapper for [Got](https://github.com/sindresorhus/got) to interact with the [GitHub API](https://developer.github.com/v3/)
 
+If you're not already using Got, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) library.
```

Comment:
How about this?

```suggestion
Unless you just need this for a single API call, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) package instead.
```

Reply:
Can you include the GraphQL package too?

Reply:
> this for a single API call

Arguably you shouldn't use got for a single call either

Reply:
```suggestion
Unless you're already using Got, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) or [@octokit/graphql.js](https://github.com/octokit/graphql.js) packages instead.
```